### ISSUE TITLE
id.gov.ua

### ISSUE THREAD

**Issue by mnaoumov** [AdguardTeam/AdguardFilters#162029]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Windows

### AdGuard version

7.14

### What type of problem have you encountered?

Website or app doesn't work properly

### Which browser(s) do you use?

Chrome

### Which device do you use?

Desktop

### Where is the problem encountered?

https://id.gov.ua/

### What filters do you have enabled?

AdGuard Base filter, AdGuard French filter, AdGuard Russian filter, AdGuard Tracking Protection filter, AdGuard Social Media filter, AdGuard Annoyances filter

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track signals

### Add your comment and screenshots

1. When AdGuard is enabled, you can't log in to https://id.gov.ua/ , you get error 900

2. Screenshots
<details><summary>Screenshot 1:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/d83bc0f7-aeb3-4691-b733-65d4ea8b4c14)

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by mnaoumov** [comment]
Based on my investigation, the login fails because the following cookies are missing: `_ga`, `_ga_8N1B6PZDBL`

The are created via JavaScript from https://www.googletagmanager.com/gtag/js?id=G-8N1B6PZDBL&l=dataLayer&cx=c , which is being blocked via AdGuard

The following rule `@@||id.gov.ua$document` solves the problem

**Comment by Alex-302** [comment]
I don't see login button on this site. Need steps to reproduce.

**Comment by mnaoumov** [comment]
@Alex-302

It is not so trivial to reproduce, because it requires some electronic key given by Ukrainian authorities.

But here are the steps

1. Connect to the VPN to have Ukrainian IP address

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/4713bbaa-62d0-439a-a6f9-227a4dfce10a)

2. Go to https://id.court.gov.ua/
3. Click `Authorize with id.gov.ua` button

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/c6e0c2c9-371f-4104-bb2e-5477e8dd787d)

4. You get redirected to https://id.gov.ua/ with some authentication query string parameters
5. Click `File media`

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/0ff7152c-5e53-45d4-ab96-4b000414df8a)

6. You get redirected to https://id.gov.ua/euid-auth-js

7. You need to provide your electronic signature key provided by Ukrainian authorities, its password and click continue

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/a94c1074-9206-4448-9442-7f717fe17603)

8. If you have AdGuard enabled, now you get the error code 900 (screenshot provided in the previous comment)

**Comment by mnaoumov** [comment]
@Alex-302 

No, what you suggested is not enough. It requires an additional rule

```
@@||googletagmanager.com/gtag/js?id=$domain=id.gov.ua
@@||www.google-analytics.com$domain=id.gov.ua
```

**Comment by Alex-302** [comment]
> @@||www.google-analytics.com$domain=id.gov.ua

It is a bit wide. We prefer make exclusins less wide as possible.

Is `@@||google-analytics.com/analytics.js$domain=id.gov.ua` enough? 
+
`@@||googletagmanager.com/gtag/js?id=$domain=id.gov.ua`

But I still cannot reproduce this problem, checked with the app and with the browser extension.

**Comment by mnaoumov** [comment]
@Alex-302 regarding steps to reproduce

Try the following additional steps

1.1. Go to https://cabinet.court.gov.ua/
1.2. Click `Enter` button

![image](https://github.com/AdguardTeam/AdguardFilters/assets/735878/0fcede6e-2671-4451-98a7-5fe4ebe6ad0a)

2. You will be redirected to https://id.court.gov.ua/

Other steps as I described before

**Comment by Alex-302** [comment]
> and even if type it manually in the rule editor, it doesn't apply either

It is not supported:( Only all userscripts can be disabled.

**Comment by mnaoumov** [comment]
@Alex-302 then the [documentation](https://adguard.com/kb/general/ad-filtering/create-own-filters) is misleading

They have an example with this exact syntax

```
@@||example.com^$extension='AdGuard Assistant'|'Popup Blocker'
```

**Comment by Alex-302** [comment]
Probably an UI bug.

Please test this:

<details>
<summary>Details</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/9c2c76cb-307a-4584-b250-15ab51e957bc)

add

`// @exclude https://*.gov.ua/*`

and save.

</details>

**Comment by slavaleleka** [comment]
> $extension modifier with specific userscript name is supported by AdGuard for Windows, Mac, and Android, running CoreLibs version 1.13 or later

AdGuard for Windows v7.14 does not use CoreLibs v1.13 yet
